### PR TITLE
Workflow doc

### PR DESCRIPTION
**Pull request by cnuland**

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -15,8 +15,7 @@ spec:
       serviceAccountName: project-initialize
       containers:
         - name: project-initialize
-          # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: "quay.io/cnuland/project-init-operator:latest"
```
Comment (sabre1041) [inline deploy/operator.yaml:18]: @cnuland did you intend to leave your personal repository as the source for the operator?

**Review by sabre1041 — COMMENTED** [review]
Seeing the following error when deploying using the manifests

```
github.com/operator-framework/operator-sdk/pkg/kube-metrics/store.go:65: Failed to list *unstructured.Unstructured: the server could not find the requested resource
```

Suggested resolution: 
https://github.com/operator-framework/operator-sdk/issues/2707

File: README.md [hunk 2]
```diff
@@ -22,45 +22,67 @@ spec:
 ```
 
 
-### Run Locally (OpenShift)
+### Install (OpenShift)
```
Comment (sabre1041) [inline README.md:11]: @cnuland I would recommend that the beginning of the README provide an overview and the goals of what the operator is looking to accomplish. The section on "Adding Defined Quota Sizes to Cluster" seems out of place

**Review by gnekic — COMMENTED** [review]
I agree with @sabre1041 comments. Other than that the updates to the README.md make sense. I'm currently spinning up a 3.x cluster to test 3.x manifests

**Review by gnekic — APPROVED** [review]
Tested defined workflow against 3.11.154 cluster, and it went smoothly

File: deploy/operator.yaml (hunk 1 above)
Comment (cnuland) [reply deploy/operator.yaml:18]: Yes, for now until we create a CI/CD pipeline for this and a COP org repo on quay. This operator is still in an early alpha, so I'm personally okay with it.

File: README.md (hunk 2 above)
Comment (cnuland) [reply README.md:11]: Cleaned it up some, added an overview section at the beginning and moved the overview of the quota below the installation

**Comment by cnuland** [comment]
@sabre1041 I wasn't able to reproduce your issue, I'm thinking it must be for a specific version of the operator-sdk? I did however apply the suggested fix, and it's still working for me, so let me know if you continue to see the issue. The README has also been reorganized to flow better.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**